server/task_executor_pitr_cutover.go

```diff
@@ -0,0 +1,73 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/bytebase/bytebase/api"
+	pluginmysql "github.com/bytebase/bytebase/plugin/db/mysql"
+	restoremysql "github.com/bytebase/bytebase/plugin/restore/mysql"
+	"go.uber.org/zap"
+)
+
+// NewPITRCutoverTaskExecutor creates a PITR cutover task executor.
+func NewPITRCutoverTaskExecutor(logger *zap.Logger) TaskExecutor {
+	return &PITRCutoverTaskExecutor{
+		l: logger,
+	}
+}
+
+// PITRCutoverTaskExecutor is the PITR cutover task executor.
+type PITRCutoverTaskExecutor struct {
+	l *zap.Logger
+}
+
+// RunOnce will run the PITR cutover task executor once.
+func (exec *PITRCutoverTaskExecutor) RunOnce(ctx context.Context, server *Server, task *api.Task) (terminated bool, result *api.TaskRunResultPayload, err error) {
+	exec.l.Info("Run PITR cutover task", zap.String("task", task.Name))
+
+	// Currently api.TaskDatabasePITRCutoverPayload is empty, so we do not need to unmarshal from task.Payload.
+
+	return exec.pitrCutover(ctx, task, server)
+}
+
+func (exec *PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.Task, server *Server) (terminated bool, result *api.TaskRunResultPayload, err error) {
+	driver, err := getAdminDatabaseDriver(ctx, task.Instance, "", exec.l)
+	if err != nil {
+		return true, nil, err
+	}
+	defer driver.Close(ctx)
+
+	instance := task.Instance
+	database := task.Database
+
+	issue, err := getIssueByPipelineID(ctx, server.store, exec.l, task.PipelineID)
+	if err != nil {
+		return true, nil, err
+	}
+
+	mysqlDriver, ok := driver.(*pluginmysql.Driver)
+	if !ok {
+		exec.l.Error("failed to cast driver to mysql.Driver", zap.Stack("stack"))
+		return true, nil, fmt.Errorf("[internal] cast driver to mysql.Driver failed")
+	}
+	mysqlRestore := restoremysql.New(mysqlDriver)
+
+	exec.l.Debug("Start swapping the original and PITR database",
```

Comment:
should use info log to be consistent with other log in this file

Reply:
Should be explicit about the source and target database here since we are swapping. If we only mention one database, it's unclear whether it's the target or the source database.